src/components/MovieDetails.tsx

```diff
@@ -1,9 +1,108 @@
-const MovieDetails = () => {
-  return (
-    <div>
-      <p>Movie details</p>
+import { useState, useEffect } from 'react';
+
+import api from 'src/service/api';
+
+import ReadMore from './ReadMore';
+
+import '../styles/MovieDetails.scss';
+
+type MovieDetailsProps = {
+  match: { params: { id: string } }
+}
+
+const MovieDetails = ({ match }: MovieDetailsProps) => {
+
+  const [movieId, setMovieId] = useState<string>(' ');
+  const [movieData, setMovieData] = useState({
```

Comment:
I think it might be useful to add type for this complex object

Reply:
done